.ibm/images/README

```diff
@@ -1,23 +1,38 @@
-This Dockerfile creates an image to be used by the IBM Cloud pipelines
+# RHDH e2e Runner - OpenShift CI Integration
 
-Currently this image is built by hand at intervals and pushed to https://quay.io/repository/rhdh-community/rhdh-e2e-runner?tab=tags when in need of an update.
+## Overview
+The `rhdh-e2e-runner` is a container image used in **OpenShift CI**, where it is deployed via a **mirror** that fetches the image from the Quay repository.
 
-```
-SHA="$(git rev-parse --short=8 HEAD)"
-# main or release-1.y
-branch="$(git rev-parse --abbrev-ref HEAD)" 
-image="quay.io/rhdh-community/rhdh-e2e-runner:${branch}"
+## Workflow for Updating the Image
 
-# build and push
-podman build . -f Dockerfile -t "${image}-${SHA}" && podman push "${image}-${SHA}"
+### **Main Branch Workflow**
+For changes to the image in `.ibm/images/Dockerfile`, follow these steps:
+1. **Perform a Local Test:**
+   - Before updating the `Dockerfile`, perform a local test by running the e2e tests to ensure the changes are stable.
 
-# update the floating (branch-named) tag too
-skopeo --insecure-policy copy --all docker://${image}-${SHA} docker://${image} 
-```
+2. **Update the Dockerfile:**
+   - Once validated, update `.ibm/images/Dockerfile` with the required changes.
 
-In future this might be automated so that if the contents of this folder changes, a new image is pushed automatically.
+3. **GitHub Actions Build & Push:**
+   - The **GitHub Actions workflow** automatically builds and pushes the updated image to **Quay.io**.
+   - This only happens **after all tests are completed successfully**.
 
-See also:
+4. **Propagation to OpenShift CI Mirror:**
+   - After the image is pushed to Quay, OpenShift CI **does not** immediately use the new image.
+   - There is a delay while OpenShift updates its **mirror** with the new version.
 
-* https://quay.io/repository/rhdh-community/rhdh-e2e-runner?tab=info
-* https://issues.redhat.com/browse/RHIDP-6244
+5. **Merge to Main Branch:**
+   - The image update process happens **before merging into `main`**.
+   - Ensure that the updated image has been successfully used in e2e tests before merging.
+
+### **Other Branches Workflow**
+For branches **other than `main`**, OpenShift CI does **not** fetch from Quay:
+- Instead, **OpenShift CI builds the image directly** from `.ibm/images/Dockerfile`.
```

Comment:
why do we have images tagged for the `release-1.y` branches @ https://quay.io/repository/rhdh-community/rhdh-e2e-runner?tab=tags if we're not going to use them? Why not benefit from prebuilt images for all the other branches too?

Reply:
@nickboldt First, we ran a test only on the main branch to see how it would behave. That's why it's currently configured only for the main branch. I will update it for other branches on a future occasion.